shared/nm-utils/nm-secret-utils.c

```diff
@@ -17,6 +17,7 @@
  * Boston, MA 02110-1301 USA.
  *
  * (C) Copyright 2018 Red Hat, Inc.
+ * (C) Copyright 2019 Jason A. Donenfeld <Jason@zx2c4.com>
```

Comment:
The file you took the function from had:

```
 * Copyright (C) 2015-2019 Jason A. Donenfeld <Jason@zx2c4.com>. All Rights Reserved.
```

IANAL and don't know particulars on if it's ever appropriate to change dates, but in my experience, I usually just copy the copyright line wholesale.

Reply:
OK, adjusted. Hope it's right (only adjusted for style -- all our such lines have the `(C)` after Copyright and a space between the year).